### PR TITLE
[INFRA] Add gcc-10 -std=c++2a support; fixes seqan/product_backlog#61

### PR DESCRIPTION
**Pull request by marehr**
Part of  seqan/product_backlog#61

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -44,6 +44,12 @@ jobs:
         - BUILD=unit
         - BUILD_TYPE=Release
         - CXXFLAGS="-std=c++17 -fconcepts"
+    - name: "Unit gcc10 (c++2a)"
```
Comment (eseiler) [inline .travis.yml:47]:
```suggestion
    - name: "Unit gcc10 (c++20)"
```
?

Comment (marehr) [reply .travis.yml:47]: It is technically only C++2a, because the standard wasn't finalised, yet.

Comment (eseiler) [reply .travis.yml:47]: Ok, was wondering, because you can pass `-std=c++20` to GCC10

Comment (marehr) [reply .travis.yml]: Okay, if it is like that we use -std=c++20 instead :)

**Comment by marehr** [comment]
@eseiler Thank you for figuring out what caused the problem.

**Pull request merged**